### PR TITLE
:book: Simulation Test Document

### PR DESCRIPTION
**Pull request by yanmxa**
## Summary

Tidy the document with different scenarios.

## Related issue(s)

Fixes #

### CODE REVIEW

File: doc/simulation/README.md [hunk 1]
```diff
@@ -13,96 +13,25 @@ There are several points we need to consider when simulating the global hub at l
...
-
-- Multicluster Global Hub Postgres
-![Global Hub Postgres CPU](./../images/global-hub-postgres-cpu-usage.png)
-![Global Hub Postgres Memory](./../images/global-hub-postgres-memory-usage.png)
-
-  | Postgres | Request | Limit |
-  |---|---|---|
-  | CPU | 0.5 | 5 |
-  | Memory | 128Mi | 4Gi |
+- [Initialize 6,000 clusters to Global Hub](./scenario/Scenario1:%206k_clusters.md)
```
Comment (clyang82) [inline doc/simulation/README.md:32]: the link is broken.

File: doc/simulation/scenario/Scenario2: 1500_clusters_150_policies.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Scenario 2: Manage 1500 clusters and 150 policies(75,000 replicated policies)
+
+## Scale
+
+- 5 Managed Hubs, Each with 300 Managed Clusters, 50 Policies
+- 1500 Managed Clusters
+- 150 Policies, 75,000 Replicated Policies
```
Comment (clyang82) [inline doc/simulation/scenario/Scenario2: 1500_clusters_150_policies.md:7]: 250 policies or 150 policies?

File: doc/simulation/README.md (hunk 1 above)
Comment (yanmxa) [reply doc/simulation/README.md]: Done.

File: doc/simulation/scenario/Scenario2: 1500_clusters_150_policies.md (hunk 2 above)
Comment (yanmxa) [reply doc/simulation/scenario/Scenario2: 1500_clusters_150_policies.md]: `250`
image

File: doc/simulation/scenario/Scenario2: 1500_clusters_150_policies.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
+# Scenario 2: Manage 1500 clusters and 150 policies(75,000 replicated policies)
```
Comment (clyang82) [inline doc/simulation/scenario/Scenario2: 1500_clusters_150_policies.md:1]: s/150/250

File: doc/simulation/README.md [hunk 4]
```diff
@@ -13,96 +13,25 @@ There are several points we need to consider when simulating the global hub at l
...
-- Multicluster Global Hub Postgres
-![Global Hub Postgres CPU](./../images/global-hub-postgres-cpu-usage.png)
-![Global Hub Postgres Memory](./../images/global-hub-postgres-memory-usage.png)
-
-  | Postgres | Request | Limit |
-  |---|---|---|
-  | CPU | 0.5 | 5 |
-  | Memory | 128Mi | 4Gi |
+- [Initialize 6,000 clusters to Global Hub](./scenario/Scenario1:%206000_clusters.md)
+- [Manage 1,500 clusters and 150 policies(75,000 replicated policies)](./scenario/Scenario2:%201500_clusters_150_policies.md)
```
Comment (clyang82) [inline doc/simulation/README.md:33]: s/150/250/

Comment (yanmxa) [reply doc/simulation/README.md]: Done.

File: doc/simulation/scenario/Scenario2: 1500_clusters_150_policies.md (hunk 3 above)
Comment (yanmxa) [reply doc/simulation/scenario/Scenario2: 1500_clusters_150_policies.md]: Done.

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**